backend/app/running_config/util/device_config_types.py

```diff
@@ -27,6 +29,9 @@ class DeviceType(str, Enum):
     SWITCH = 'SWITCH'
     UNKNOWN = 'UNKNOWN'
 
+    def to_netmiko_device_type(self):
+        return NetmikoDeviceType.CISCO_IOS if self.value != self.UNKNOWN else NetmikoDeviceType.UNKNOWN
+
```

Comment:
A jeśli dojdą urządzenia jakiegoś innego typu? https://github.com/ktbyers/netmiko/blob/develop/PLATFORMS.md

Reply:
To wtedy config nie będzie robiony w PacketTracer i pół kodu będzie do wymiany. Poza tym na podstawie czego mamy to brać innego, jak nie z pola DeviceType? Na razie działa.